core/constraint-streams/src/main/java/ai/timefold/solver/constraint/streams/common/quad/QuadConstraintBuilderImpl.java

```diff
@@ -32,7 +32,8 @@ protected PentaFunction<A, B, C, D, Score_, ConstraintJustification> getJustific
     public <ConstraintJustification_ extends ConstraintJustification> QuadConstraintBuilder<A, B, C, D, Score_> justifyWith(
             PentaFunction<A, B, C, D, Score_, ConstraintJustification_> justificationMapping) {
         if (this.justificationMapping != null) {
-            throw new IllegalStateException("Justification mapping already set (" + justificationMapping + ").");
+            throw new IllegalStateException("Justification mapping already set (" + justificationMapping
+                    + ").\nMaybe the constraint calls justifyWith() twice?");
```

Comment:
Please convert to the new convention, using multi-line strings and placeholders.